client/components/Home.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
```

Comment:
Linting standards are inconsistent. Single quotes in previous file, double quotes here. Please choose one and stick with it.